src/main/java/core/basesyntax/AreaCalculate.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaCalculate {
```

Comment:
Don't use verbs for class/interface names. It is better to name the interface as a noun that describes its capability, like `AreaCalculator` instead of `AreaCalculate`. Refer to the checklist for an example of a good interface name.